ts/parts/ColumnSeries.ts

```diff
@@ -1066,9 +1066,12 @@ seriesType<Highcharts.ColumnSeries>(
 
                     if (graphic) { // update
                         graphic[verb](
-                            merge(shapeArgs)
+                            merge(shapeArgs),
+                            null,
+                            function (): void {
+                                H.fireEvent(point, 'afterPointAnimate');
```

Comment:
Unfortunately, I cannot change the `shapeArgs` in the translate override for polar the same way as the graphic is updated on `afterPointAnimate` because then points would animate directly to the center of the pane (like in the following demo after hiding the Series 2: http://jsfiddle.net/BlackLabel/ycrb6qf5/). As for using one of the existing events, the closest one is the `afterRender` at the end of the series' render function but that would result in points simply disappear (the first three points: http://jsfiddle.net/BlackLabel/xv04jby1/). Could we maybe instead of firing the `afterPointAnimate` event for every column point, make that every radial bar point would have reference to a custom complete function and the call of the animate function in the `drawPoints` would look like the following: `graphic[verb](merge(shapeArgs), void 0, point.complete)`. This way the `drawPoints` function would run just as before for regular column points but in case of radial bar points the `complete` function would be added and resulting in a correct animation process.